tests/test_driver.py

```diff
@@ -185,37 +185,69 @@ def test_get_outputs(self, driver, carol_pubkey,
             'output_index': 0
         } in outputs
 
-    @mark.parametrize('spent,outputs_qty', ((False, 1), (True, 1), (None, 2)))
-    def test_get_outputs_with_spent_query_param(
-            self, spent, outputs_qty, driver, carol_pubkey,
-            persisted_carol_bicycle_transaction,
-            persisted_carol_car_transaction,
-            persisted_transfer_carol_car_to_dimi):
-        outputs = driver.outputs.get(carol_pubkey, spent=spent)
-        assert len(outputs) == outputs_qty
-
-        # Return only unspent outputs
-        if spent is False:
-            assert {
-                'transaction_id': persisted_carol_bicycle_transaction['id'],
-                'output_index': 0
-            } in outputs
-        # Return only spent outputs
-        elif spent is True:
-            assert {
-                'transaction_id': persisted_carol_car_transaction['id'],
-                'output_index': 0
-            } in outputs
-        # Return all outputs for carol
-        elif spent is None:
-            assert {
-                'transaction_id': persisted_carol_bicycle_transaction['id'],
-                'output_index': 0
-            } in outputs
-            assert {
-                'transaction_id': persisted_carol_car_transaction['id'],
-                'output_index': 0
-            } in outputs
+    def test_get_outputs_with_spent_query_param(self, driver):
+        from bigchaindb_driver.crypto import generate_keypair
+        import uuid
+
+        outputs = [(False, 2, 1), (True, 0, 1), (None, 2, 2)]
+
+        def create_asset():
```

Comment:
The name of this function (`create_asset()`) is a bit misleading because it prepares a CREATE transaction, it doesn't really create an asset. To create an asset, I think you'd also have to fulfill the CREATE transaction, post the result to a BigchainDB network, and get it committed to the blockchain. A name like `prepare_create_tx()` would be more accurate.

Reply:
ah, ja.. I named it when it was still just creating an asset and forgot to rename it later. I'll change that :)